RxJavaTemplate/domain/build.gradle

```diff
@@ -44,6 +44,12 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    android {
+        lintOptions {
+            xmlReport true
```

Comment:
hmm... why the previous config for github action did not have this lint result attached as well? 🤔

Reply:
Ah because I remove the usage of the [danger-kotlin-android-lint-plugin](https://github.com/gianluz/danger-kotlin-android-lint-plugin) as it was not working properly and thus this lint result attachment is not needed. 

Now since we all agreed to use danger-ruby for both CIs, let me update to attach this lint result for the Github Action 🙏